conf/locale/he/LC_MESSAGES/django.po

```diff
@@ -11300,11 +11300,17 @@ msgid "You should Register before trying to access the Unit"
 msgstr "עליך להירשם לפני שתוכל לגשת ליחידה"
 
 #: lms/templates/enroll_staff.html lms/templates/ccx/enrollment.html
-#: lms/templates/courseware/course_about.html
 #: lms/templates/instructor/instructor_dashboard_2/membership.html
+msgctxt "someone"
 msgid "Enroll"
 msgstr "הירשם"
 
```

Comment:
I added anyway, but... this is not redundant? I mean, Enroll string is the default enroll translation?

Reply:
When you execute the extraction command with i18n_tools, does this appear? or did you create it manually?

Reply:
I added it manually

Reply:
that means it does not appear when you extract it? Does the `+msgctxt "self"` from line 14144 appear when you extract?

Reply:
No, neither. Both I added manually. The process that I did: Do the change in the template. Run `paver i18n_extract`, add manually the strings in .po file. Run `paver i18n_fastgenerate`. If I extract the strings and run `fastgenerate` it does not add the `msgctxt`by itself.